.github/workflows/build-test.yaml

```diff
@@ -50,15 +50,13 @@ jobs:
           /o:"qowaiv" \
           /d:sonar.token="${{ secrets.SONAR_TOKEN }}" \
           /d:sonar.host.url="https://sonarcloud.io" \
-          /d:sonar.cs.opencover.reportsPaths="**/coverage.**.opencover.xml" \
+          /d:sonar.cs.opencover.reportsPaths="**/coverage.opencover.xml" \
           /d:sonar.cs.vstest.reportsPaths="**/*.trx"
       - name: Test
         run: |
           dotnet test specs/Qowaiv.Specs --configuration Release \
           --logger "trx;LogFileName=test-results.trx" \
-          /p:CollectCoverage=true \
-          /p:ThresholdType=branch \
-          /p:CoverletOutputFormat=opencover
+          --collect:"XPlat Code Coverage;Format=opencover"
```

Comment:
What is the meaning of this `XPlat Code Coverage`?

Reply:
Apparently it's the term to indicate the use `coverlet.collector`, see https://www.github.com/coverlet-coverage/coverlet/issues/1065.